packages/fxa-react/components/LinkExternal/index.tsx

```diff
@@ -4,13 +4,18 @@
 
 import React from 'react';
 import { Localized } from '@fluent/react';
+import { GleanDataType2fa } from '../../lib/interfaces';
 
 type LinkExternalProps = {
   className?: string;
   href: string;
   children: React.ReactNode;
   title?: string;
   'data-testid'?: string;
+  gleanDataAttrs?: {
+    id: string;
+    type?: GleanDataType2fa;
```

Comment:
Seems strange to me that a 2fa specific type is used in a much more generic type for link props.

Reply:
Sure, but wouldn't we just expand it at that time? That's what I pictured but, I can change to 'string'. (Maybe this should be an enum anyways...)

Reply:
> but wouldn't we just expand it at that time?

Does it make sense to have 'inline setup' always as a valid value of that type when applied to LinkExternal?  I'm arguing no and we might as well be generic.  I could see using a type argument to limit the valid values to whatever context in which the LinkExternal is used.

Reply:
Eh yeah good point. Mostly I just wanted us to not typo the `type` here but I'll wait for #17459 and #17460 to land and then I'll adjust this and turn that 2fa type into an enum and update all around, will likely request re-review.